src/vm/method.cpp

```diff
@@ -4994,24 +5000,29 @@ LPVOID NDirectMethodDesc::FindEntryPoint(HINSTANCE hMod) const
     DWORD nbytes = (DWORD)(strlen(GetEntrypointName()) + 1);
```

Comment:
Instead of `GetEntrypointName()` we should use the `funcName` local or remove the variable entirely.